content/release-10-16-0/devices-10-16-0-bundle/10-16-0-0.md

```diff
@@ -76,6 +76,18 @@ LWM2M</td>
 DM-1336</td>
 </tr>
 
+<tr>
+<td>LWM2M</td>
+<td>When the client sent data to the LWM2M agent for an unmapped event the agent triggered an alarm instead of creating an event. Now the LWM2M agent creates a <code>c8y_lwm2m_sendData</code> event. Otherwise, if there is already a mapping uploaded to the agent, the data is processed by the mapping engine.</td>
+<td>DM-373</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>LWM2M 1.1 allows direct writing to a resource instance. That means if you have a resource, which has some resource instances (as an example /3/0/11/0) and these resource instances allows direct write, it's possible to write values to any of the resource instances.</td>
```

Comment:
```suggestion
<td>LWM2M 1.1 allows direct writing to a resource instance. That means if you have a resource, which has some resource instances (for example, /3/0/11/0) and these resource instances allow direct writing, it's possible to write values to any of the resource instances.</td>
```
i don't quite understand the example given here. it could probably also use some formatting, it looks like a special string.